build.gradle

```diff
@@ -132,6 +132,8 @@ dependencies {
 
     // Compile-Time Dependencies
     annotationProcessor "com.github.bsideup.jabel:jabel-javac-plugin:${jabel_version}"
+    // workaround for https://github.com/bsideup/jabel/issues/174
+    annotationProcessor "net.java.dev.jna:jna-platform:5.13.0"
```

Comment:
It would be nice to keep the version number in `build.properties`, so it's consistent with the rest.

Reply:
Added